boards/sim/sim/sim/scripts/Make.defs

```diff
@@ -63,6 +63,11 @@ ARCHINCLUDES = -I. -isystem $(TOPDIR)/include
 ARCHINCLUDESXX = -I. -isystem $(TOPDIR)/include -isystem $(TOPDIR)/include/cxx
 ARCHSCRIPT =
 
+ifneq ($(KERNEL),y)
```

Comment:
Why skip -fno-common for KERNEL build?

Reply:
maybe a mistake. i'll check.

Reply:
i don't remember why i did the ifneq. i removed the conditional as it's necessary regardless of KERNEL.